webapp/pages/post/_id/_slug/index.vue

```diff
@@ -179,6 +179,20 @@ export default {
       if (!author) return false
       return this.$store.getters['auth/user'].id === author.id
     },
+    sortedTags() {
```

Comment:
I would love to have this imported as a more general function located in `components/utils/PostHelpers` or even in a more general utility file.
This way it can be reused in `webapp/pages/post/_id/_slug/more-info.vue`. 👇🏼

It can be used to sort the post categories in the future. 🙏🏼
A missing issue as far as I see …